rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,184 @@
+
+=== Elections ===
+
+The election process takes place annually, concurrent with the election of the Release Managers in approximately
+March/April.  (The exact date may vary year to year.)  The sitting TC Secretary officially begins the process with
+a call for nominations.  The nomination period lasts for one week.
+
+During that week, any eligible core developer may self-nominate by posting to the Internals mailing list.  If the
+candidate is paid to work on php-src or PHP extensions, they must state in their nomination which company or organization
+they work for.  Failure to do so will result in disqualification, as determined by the TC Secretary.
+
+Once nominations are over, the voting process will begin.  Eligible voters are the same as for any RFC.
+
+If there are five or fewer nominees, there is no vote and all nominated individuals are elected.
+
+If there are six or more nominees, a two week vote using the Single Transferable Vote method will take place.  The five
+winning candidates at the end of the vote are elected.
+
+The newly elected TC members will elect from their own number a TC Secretary by simple majority.
+
+=== Vacancies ===
+
+Any TC member may resign at any time via an email to the Internals mailing list.  If the resigning member is the TC
+Secretary, then the TC will elect a new Secretary from its number.
+
+The TC may declare one of its members AWOL after no communication from that individual for one month, by a majority vote.
+If the vote passes, the TC member is removed.
+
+If the number of TC members is reduced to three or fewer and there are more than two months until the next regularly
+scheduled election, the Secretary must call for a special election for the vacant seats.  The election process is the
+same as for any other election, but the newly elected TC members will serve only until the next regular election.
+
+Should the number of TC members fall to zero, the senior-most Release Manager of the latest stable release will step
+ in to act as TC Secretary and immediately call for a new vacancy election.
+
+==== Work flow ====
+
+The TC is primarily a reactive body, and is not expected to proactively search PRs for issues.  However, TC members
+are permitted to comment on any open issue, and contributors are encouraged to give their comments appropriate weight.
+
+Should a dispute or question about a change arise, the TC may be called on to resolve it.
+
+  - Any core developer (including a TC member) may ask for TC resolution of an issue by adding the label `tc-hold` to it and @-mentioning the TC GitHub team.
+  - An issue with the `tc-hold` tag MUST NOT be merged.
+  - The TC members will confer in whatever manner they deem most convenient, public or private or both.
+    * The TC SHOULD consult with relevant other individuals, including the core developers involved in the issue, Release
+      Managers, and impacted extension maintainers as appropriate.
```

Comment:
These two lines don't render right in the wiki page.

Reply:
Fixed